### ISSUE TITLE
Some badges can be uploaded multiple times

### ISSUE THREAD

**Issue by ghost** [mozilla/openbadges-backpack#486]
This does not happen with Badges 101, Web Navigator or Webmaker badges.

The ones I can do this with are from:
http://www.codeschool.com/
http://www.itap.purdue.edu/studio/passport/

![Screen Shot 2013-01-20 at 4 10 41 AM](https://f.cloud.github.com/assets/731748/80594/6f70e8fe-6274-11e2-8325-9b53a5b8c697.png)

**Comment by mattdigitalme** [comment]
@gideonthomas Is this still the case - I couldn't seem to push a Code School badge twice - can you check again and if fixed we'll close this issue @auralon

**Comment by auralon** [comment]
@mattdigitalme this is still an issue, as a Codeschool assertion url generates a new hash for every request.  This is more an issue their end than ours.  We could extend the validation to capture such circumstances, but I'm not sure it is entirely required.

**Comment by mattdigitalme** [comment]
OK cool @auralon also jsut refreshed my Backpack - 2 CS badges present!

**Comment by auralon** [comment]
Yeah, Codeschool could really benefit from using the BP Connect API anyhow.
